app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java

```diff
@@ -101,6 +101,11 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         prefs = PreferenceManager.getDefaultSharedPreferences(getActivity());
         license = prefs.getString(Prefs.DEFAULT_LICENSE, Prefs.Licenses.CC_BY_SA_3);
 
+        // check if this is the first time we have uploaded
+        if (prefs.getString("Title", "").trim().length() == 0 && prefs.getString("Desc", "").trim().length() == 0) {
```

Comment:
The style checker says this line is too long. Can you perhaps split it at the && operator?